### DISCUSSION TITLE
React Native support

### DISCUSSION THREAD

**Discussion by vsamerv** [microsoft/FluidFramework#15095] [General]
Hello, I am trying to utilize Fluid Framework between Web App (Angular) and Mobile App (React Native). Is it even possible to use Fluid Framework with React Native? Thank you.

**Comment by vsamerv** [comment]
Asked in Q&A

**Comment by imvikysharma** [comment]
Yes, it is possible to use Fluid Framework between a web app built with Angular and a mobile app built with React Native. Fluid Framework is designed to work across different platforms and languages, allowing you to share collaborative data and experiences between different apps and devices.

To use Fluid Framework with React Native, you can use the @fluidframework/react-native package, which provides a set of components and hooks for building Fluid-enabled React Native apps. This package is still in preview, but you can find more information and examples on the official Fluid Framework documentation and GitHub repository.

I would like to recommend a book on React

https://www.amazon.com/dp/B0BVRKSY6G?maas=maas_adg_A78FB1C22F0F576564CF29D2524311F0_afap_abs&ref_=aa_maas&tag=maas

**Reply by vsamerv** [reply]
Hello,

Thank you for the answer. However, I am not able to find @fluidframework/react-native package anywhere, can you please provide the link? Or you are saying that syntax is pretty much the same as React?

This React book looks interesting, I will consider to purchase it.
